lib/math_helper.ex

```diff
@@ -59,4 +59,41 @@ defmodule MathHelper do
   """
   @spec log(number(), number()) :: number()
   def log(x, b), do: :math.log(x) / :math.log(b)
+
+  @doc """
+  Returns the byte size of an integer
+
+  ## Examples
+
+      iex> MathHelper.integer_byte_size(0)
+      0
+
+      iex> MathHelper.integer_byte_size(1)
+      1
+
+      iex> MathHelper.integer_byte_size(0xfffffffff)
+      5
+
+  """
+  @spec integer_byte_size(number()) :: number()
+  def integer_byte_size(n) when n == 0, do: 0
+  def integer_byte_size(n), do: byte_size(:binary.encode_unsigned(n))
+
+  @doc """
+  Bits to bytes
+
+  ## Examples
+
+      iex> MathHelper.bits_to_bytes(0)
+      0
+
+      iex> MathHelper.bits_to_bytes(9)
+      1
+
+      iex> MathHelper.bits_to_bytes(256)
+      8
+
+  """
+  @spec bits_to_bytes(number()) :: number()
+  def bits_to_bytes(n), do: round(:math.ceil(n / 32))
```

Comment:
Shouldn't this be `n / 8`, not `n / 32`?

Reply:
(or maybe this is `bits_to_words`?)

Reply:
Ah, yep! It's `bit_to_words`: [Renamed!](https://github.com/exthereum/evm/pull/15/commits/c39f48f89e5cb0ffc6f749774a6c038498cb6a62)